lib/logic/triggers.js

```diff
@@ -60,14 +60,12 @@ LineCommentsTrigger.TYPE = "line_comments";
 
 class PullRequestListTrigger extends Trigger {
     static TYPE: string;
-    myPulls: Array<PullRequest>;
-    assignedPulls: Array<PullRequest>;
+    pulls: Object;
     error: ?string;
 
-    constructor(myPulls: Array<PullRequest>, assignedPulls: Array<PullRequest>, error: ?string) {
+    constructor(pulls: Object, error: ?string) {
```

Comment:
When you say `Object` you don't *really* mean that, you mean it more as a record type. Can we please represent it as such? E.g.

```javascript
constructor(pulls: {assigned: { needsAction: PullRequest[], othersInvolved: PullRequest[], noAction: PullRequest[] } }, error: ?string) {
```

Of course this is getting unruly, so it might be worth either:
 - Splitting it up in the constructor args as it was before.
 - Pass in an array of `PullRequest` and let the trigger bucket them.

I'm thinking the latter.